carbonmark/components/pages/Retirements/RetirementProvenance/RetirementCard/index.tsx

```diff
@@ -36,10 +36,13 @@ const RetirementCardItem = (props: {
       <Text t="body3" color="lightest" uppercase={true}>
         {props.title}
       </Text>
-      <Text t="body1" color="lightest" className={styles.iconAndText}>
+      <div className={styles.iconAndText}>
+        {" "}
         {props.icon}
-        {props.text}
-      </Text>
+        <Text t="body1" color="lightest">
+          {props.text}
+        </Text>
```

Comment:
> the profile logo was placed inside a <Text> component which caused a hydration error

Wait I'm confused.. it *was* contained inside a text element or it *should* be included inside one?..

Reply:
The icon was previously inside the <Text> component which caused the hydration error:
https://github.com/KlimaDAO/klimadao/pull/2244/files#diff-b601e880a770e0f7f156b6a18d097975761c04440b92660adc73b82b4c08de4fL39-L42

Reply:
But you're adding it inside a `<Text` component here in this pr..?

Reply:
When `props.icon` was inside the `<Text>` component with `props.text`:
``` typescript
    <Text t="body1" color="lightest" className={styles.iconAndText}>
          {props.icon}
          {props.text}
    </Text>
```

there was a hydration error. By moving `props.icon` outside of the `<Text>` component:
```typescript
      <div className={styles.iconAndText}>
        {props.icon}. <-----
        <Text t="body1" color="lightest">
          {props.text}
        </Text>
```
The hydration error seems to be resolved(?).

Reply:
Hmm ok, fair enough